knotx-knot/knotx-knot-service/src/main/java/io/knotx/knot/service/service/ServiceEngine.java

```diff
@@ -83,7 +85,9 @@ private JsonObject buildResultObject(AdapterResponse adapterResponse) {
     } else if (rawData.charAt(0) == '{') {
       object.put(RESULT_NAMESPACE_KEY, new JsonObject(rawData));
     } else {
-      throw new DecodeException("Result is neither Json Array nor Json Object");
+      LOGGER.error("Result of [{} {}] neither Json Array nor Json Object: [{}]",
```

Comment:
Should we hide this exception here? It changes a service error scenario behaviour.

Reply:
We're not hiding anything here ? Before it was the exception generated here that broke the whole processing. But in fact instead it should log the error that data is not parsable and return empty json.